README.md

```diff
@@ -24,8 +24,23 @@ For details on the new smoke test strategy and contributor guidance, see [TESTIN
 
 ## <a name="about"></a>About
 
-**kleinanzeigen-bot** is a console-based application to simplify the process of publishing ads on kleinanzeigen.de.
-It is a spiritual successor to [Second-Hand-Friends/ebayKleinanzeigen](https://github.com/Second-Hand-Friends/ebayKleinanzeigen).
+**kleinanzeigen-bot** is a command-line application to **publish, update, delete, and republish listings** on kleinanzeigen.de.
+
+### Key Features
+- **Automated Publishing**: Publish new listings from YAML/JSON configuration files
+- **Smart Republishing**: Automatically republish listings at configurable intervals to keep them at the top of search results
+- **Bulk Management**: Update or delete multiple listings at once
+- **Download Listings**: Download existing listings from your profile to local configuration files
+- **Browser Automation**: Uses Chromium-based browsers (Chrome, Edge, Chromium) for reliable automation
+- **Flexible Configuration**: Configure defaults once, override per listing as needed
+
+### Why This Project?
+
+This is a complete Python rewrite of [Second-Hand-Friends/ebayKleinanzeigen](https://github.com/Second-Hand-Friends/ebayKleinanzeigen). The rewrite provides:
```

Comment:
we used to have a section like this but I removed it in 3d27755207e8433bfdc36e490c39ab5dc7d42ca1 as I believe this is not really interesting for end users